repository-data/application/src/main/resources/hcm-config/hst/configurations/nationalindicatorlibrary/components/nihub.yaml

```diff
@@ -0,0 +1,6 @@
+---
+definitions:
+  config:
+    /hst:hst/hst:configurations/nationalindicatorlibrary/hst:components/nihub:
+      hst:componentclassname: uk.nhs.digital.nil.components.NiHubComponent
```

Comment:
You don't actually need this file. The component class is already specified in pages/nihub.yaml

Reply:
+1

Reply:
Ah, good call. I was curious quite how much boilerplate was needed but this is one less file :)